breba_app/main.py

```diff
@@ -46,7 +46,15 @@ async def lifespan(app):
 
 app_path = Path(__file__).parent
 
+import time
+DEV_MODE = os.getenv("DEV_MODE", "true").lower() == "true"
+
+def asset_url(filename):
+    base = f"/public/{filename}"
+    return f"{base}?asset_version={int(time.time())}"
```

Comment:
This will bust the cache every time. That means caching will be disabled and the site will load more slowly every time.

Reply:
Maybe as a quick shortcut we could on server startup calculate a version number(random?) and then use that for all static assets. I think this could work short term although it would mean that if we have multiple server instances, they could have different "version" numbers, but I think this problem is currently not a problem... We can later solve it with devops